build-image/first-boot.bash

```diff
@@ -129,36 +133,30 @@ echo "OK"
 echo -n "$(timestamp) [openHABian] Updating repositories and upgrading installed packages... "
 apt update &>/dev/null
 apt --yes upgrade &>/dev/null
-if [ $? -eq 0 ]; then
-  echo "OK";
-else
-  dpkg --configure -a
```

Comment:
Needed to remove this code to get it running in the VM container at Travis, don't ask me why. 
Did this code solve a specific edge case? @ThomDietrich